### ISSUE TITLE
Updating Lesson Request list (English)

### ISSUE THREAD

**Issue by acrymble** [programminghistorian/jekyll#1238]
The current lesson requests are quite old and some have been taken up by people. It currently reads:

- What can you conclude from topic models? (in progress)
- How do you conduct spatial clustering of geographic data?
- When do you know your network analysis is meaningful?
- TF-IDF to Historical Research (done)
- Space Syntax of Historical Data
- How to Publish Digital Scholarly Editions using a Native XML Database
- How to Analyse Audio Artefacts

--

Please suggest new lesson ideas for lessons we'd like people to write so I can update this. I'll do this at the end of March 2019, so please get ideas in soon.

**Comment by fdlaramee** [comment]
1. Multidimensional Scaling, when is it useful, how to do it right, how to interpret the results, and how the method’s variants differ from each other. How do principal component analysis and correspondence analysis fit within the spectrum.

2. Word vectors: how to make your own without Google’s billions of words of training data, what can be done with them. (Or: how to take advantage of Google’s models if you are working with contemporary history? Not even sure that’s possible, but probably useful if it is.)

3. An introduction to TensorFlow.js, now that it has reached 1.0 status.

4. Maybe, as a counterpart to Audacity for editing sound files, some sort of screen capture tutorial for people who want to prepare course materials, demos, that sort of thing.

5. This last one may be stretching our boundaries a little: How to set up an academic podcast, with questions of hosting, RSS feed, finding an audience, that sort of thing.

**Comment by acrymble** [comment]
I think I'd also like one on choosing appropriate metadata schema for different types of objects/sources/artefacts.

**Comment by amsichani** [comment]
I second the one on word vectors and Adam's idea on metadata schemas' selection and Metadata Αpplication Profiles.

**Comment by acrymble** [comment]
@fdlaramee I have added in the two I understood well enough to describe. I can add your other ideas if you can give me 2 sentences that explain what we're looking for, which follow this type of tone:

> Ok, so we’ve built a great network diagram. How do we move to the next step and form meaningful conclusions? This is about starting with a graph and shifting into analysis mode. If you can help our readers take that step, we want to hear from you.



1. Word vectors: how to make your own without Google’s billions of words of training data, what can be done with them. (Or: how to take advantage of Google’s models if you are working with contemporary history? Not even sure that’s possible, but probably useful if it is.)
2. An introduction to TensorFlow.js, now that it has reached 1.0 status.
3. Maybe, as a counterpart to Audacity for editing sound files, some sort of screen capture tutorial for people who want to prepare course materials, demos, that sort of thing.

**Comment by acrymble** [comment]
@spapastamkou @fdlaramee @mpuren maybe you would like to create a similar page for lessons you're looking for someone to translate. That's a separate issue, but worth considring.

I'm closing this now. @fdlaramee please feel free to reopen if you want me to add those other 3.
